src/test/kotlin/no/nav/familie/ef/sak/repository/DomainUtil.kt

```diff
@@ -103,6 +103,30 @@ fun tilkjentYtelse(behandlingId: UUID, personIdent: String): TilkjentYtelse = Ti
                                     samordningsfradrag = 0,
                                     kildeBehandlingId = behandlingId)))
 
+fun tilkjentYtelse(behandlingId: UUID,
```

Comment:
Her finnes det allerede metoder i denne fila for å lage tilkjent ytelse og i `ØkonomiUtils`, er det mulig å bruke noen av de, eller endre de for å unngå å lage flere metoder av samme typen? :)